mesheryctl/internal/cli/root/perf/view.go

```diff
@@ -0,0 +1,94 @@
+package perf
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var outFormatFlag string
+
+var viewCmd = &cobra.Command{
+	Use:     "view",
+	Short:   "view perf profile",
+	Long:    `See the configuration of your performance profile`,
+	Example: "mesheryctl perf view [ performance test profile name ]",
```

Comment:
```suggestion
	Example: "mesheryctl perf view [profile-id]",
```
We're only able to fetch profile based on profile-ids

Reply:
ok, so the issue was not written correctly ? https://github.com/layer5io/meshery/issues/2827

Reply:
Actually, that was discovered after writing that issue. :sweat_smile: